lib/assert.js

```diff
@@ -386,6 +388,35 @@ class Assertions {
 			}
 		});
 
+		this.like = withSkip((actual, likePattern, message) => {
+			if (!checkMessage('like', message)) {
+				return;
+			}
+
+			if (isPlainObject(likePattern) && Object.keys(likePattern).length === 0) {
```

Comment:
The pattern itself *must* be a plain object, otherwise folks should be using `deepEqual`.